main.go

```diff
@@ -99,6 +99,16 @@ func setupFlags(cmd *cobra.Command) {
 	// volume config
 	flagSet.StringVar(&cfg.VolumeConfig.DriverAlias, "volume-driver-alias", "", "Set volume driver alias, <name=alias>[;name1=alias1]")
 
+	// network config
+	flagSet.StringVar(&cfg.NetworkConfig.BridgeConfig.Name, "bridge-name", "", "Set default bridge name")
+	flagSet.StringVar(&cfg.NetworkConfig.BridgeConfig.IP, "bip", "", "Set bridge ip")
+	flagSet.StringVar(&cfg.NetworkConfig.BridgeConfig.GatewayIPv4, "default-gateway", "", "Set default bridge gateway")
```

Comment:
Set IP address for default bridge gateway?

In addition, when we use IP in description or document, we must use IP with both letters uppercased. Please update the block of code. 

MTU and CIDR as well.

Reply:
> Set IP address for default bridge gateway?
yes, it is bridge config, so set bridge ip address

> In addition, when we use IP in description or document, we must use IP with both letters uppercased. Please update the block of code.
>
> MTU and CIDR as well.
OK, I will modify